pandas/parser.pyx

```diff
@@ -1878,6 +1880,17 @@ cdef kh_float64_t* kset_float64_from_list(values) except NULL:
 
 
 cdef raise_parser_error(object base, parser_t *parser):
+    cdef:
+        object old_exc
+        PyObject *type, *value, *traceback
+    if PyErr_Occurred():
```

Comment:
The return from PyErr_Occurred does not need to be decrefed. The return from PyErr_Fetch does, and is.